### PR TITLE
#7 Feature/lint-action

### LINKED ISSUE #7: Lint Github action
Create a workflow that checks code in PR-s. Use ESLint & Prettier (with dedicated config files)

### PR DESCRIPTION
**Pull request by gergoradeczki**
- Added Prettier and ESLint support with dedicated config files.
- VSCode now recommends to use ESLint and Prettier extensions
- GitHub Action to check PR

### CODE REVIEW

**Review by triszt4n — CHANGES_REQUESTED** [review]
Please resolve the conflict in package-lock.json file 🧑‍🚀 

**EDIT:** Non-relevant anymore

File: .prettierrc.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+    "singleQuote": true,
+    "useTabs": false,
+    "tabWidth": 2,
+    "semi": false,
+    "endOfLine": "auto"
```
Comment (triszt4n) [inline .prettierrc.json:6]: These are nice stuff

File: .github/workflows/eslint.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
+name: eslint
+
+on:
+  pull_request:
+    branches:
+      - main
```
Comment (nemaberci) [inline .github/workflows/eslint.yml]: Shouldn't this be master?

File: .github/workflows/prettier.yml [hunk 3]
```diff
@@ -0,0 +1,28 @@
+name: prettier
+
+on:
+  pull_request:
+    branches:
+      - main
```
Comment (nemaberci) [inline .github/workflows/prettier.yml]: Same here with master/main

**Review by nemaberci — CHANGES_REQUESTED** [review]

File: .github/workflows/eslint.yml (hunk 2 above)
Comment (gergoradeczki) [reply .github/workflows/eslint.yml]: yes

Comment (gergoradeczki) [reply .github/workflows/eslint.yml]: done

File: .github/workflows/prettier.yml (hunk 3 above)
Comment (gergoradeczki) [reply .github/workflows/prettier.yml]: done

**Comment by gergoradeczki** [comment]
Please squash merge this PR to keep timeline clean

**Review by nemaberci — APPROVED** [review]
😎

**Comment by gergoradeczki** [comment]
@triszt4n 🥺👉👈

**Review by triszt4n — APPROVED** [review]
Yep, all good

**Review by bbucsy — APPROVED** [review]

**Pull request merged**